# MotorControlService.srv
turntable_control/MotorControl request
---
bool response